### ISSUE TITLE
add option to cloak/hide auto-suggestions

### ISSUE THREAD

**Issue by Norfeldt** [dotenv-org/dotenv-vscode#68]
First: 👏👏👏

This extension is fantastic. Thank you so much 🙏

The cloak mode is also a great option. However it's kinda problematic that it shows the values when auto-completing:

![Screenshot 2022-11-08 at 09 06 10](https://user-images.githubusercontent.com/678806/200508822-40bea306-2193-460f-a4f0-c00a4534d93b.png)

I would love if it was doing something in between like `development` becomes `*...*nt`. 

```ts
export function hideString(str: string | undefined) {
  if (!str) return typeof str
  const hideChars = str.length > 8 ? 2 : 1
  const re = new RegExp(`.*(?=.{${hideChars}}$)`, 'i')

  return str.replace(re, '*...*')
}
```

**Comment by motdotla** [comment]
Good point. @Norfeldt are you using it for screen sharing? Is that why it is important for you to also cloak while auto-completing?

**Comment by Norfeldt** [comment]
exactly @motdotla 👍

**Comment by TheRealSmeddy** [comment]
Would be nice to see this feature in upcoming updates.

**Comment by motdotla** [comment]
We will make this is happen soon.

**Comment by motdotla** [comment]
This is now available in `0.26.0`. https://github.com/dotenv-org/dotenv-vscode/pull/95

Go to your Dotenv extension settings and toggle autopeeking to off.
